.github/workflows/traiage.yaml

```diff
@@ -134,7 +160,11 @@ jobs:
         if: inputs.cleanup
         env:
           BUCKET_PREFIX: "gs://coder-traiage-outputs/traiage"
+          CODER_USERNAME: ${{ steps.get-coder-username.outputs.coder_username }}
```

Comment:
Thoughts about echoing into the env file rather than pulling from output?

Reply:
I'm mostly indifferent, but I think explicitly pulling from output makes it easier to see the dependency chain between the steps. If you just dump everything into `GITHUB_ENV` it's easier to mix up the order imo.